suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml

```diff
@@ -157,3 +157,13 @@ tests:
         config-file-name: test_multitenant_access.yaml
         timeout: 300
     desc: Perform rgw tests
+
+- test:
+    name: validate-rgw-pool
+    module: test_validate_osd_pool.py
+    polarion-id: CEPH-83575014
+    config:
+      cmd: ceph osd pool ls detail
+      sudo: True
+    desc: Validate osd pool after upgrade
```

Comment:
The test is specific to verifying if `.rgw.root` data pool... I believe we can accomplish this by using the following lines
```
- test:
    name: test non default rgw pool creation
    module: exec.py
    polarion-id: 
    config:
      check_status: true
      cephadm: true
      commands:
        - "ceph osd pool ls detail | grep -v .rgw.root"
    desc: Verifies if default rgw pool is created when RGW service is deployed.
```